src/olympia/lib/akismet/tasks.py

```diff
@@ -0,0 +1,10 @@
+from olympia.amo.celery import task
+
+
+@task
+def submit_to_akismet(report_ids, submit_spam):
+    from .models import AkismetReport  # circular import
+
+    reports = AkismetReport.objects.filter(id__in=report_ids)
+    for report in reports:
+        report.submit_spam() if submit_spam else report.submit_ham()
```

Comment:
What's the error behavior here, in case of, say a network error we ignore the exception, log something in Kibana and try again whenever that task is run again?

Reply:
Currently it just gets logged in kibana as an error if the `response.content` isn't exactly "Thanks for making the web a better place.".  There isn't any automatic retry and the task is only (currently) fired via someone using the action in django admin.  

In some ways that's not ideal - intermittent network issues would be mean the action is ignored; but in others it's normal behaviour on AMO - we don't generally retry any tasks, and django admin would reflect that it hadn't been submitted.  I wasn't sure what was needed here so went with the easy option (do nothing)

Reply:
> the task is only (currently) fired via someone using the action in django admin.

As long as the task is being run if the action is being run manually I'm all fine with this.